index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <script src="https://kit.fontawesome.com/bf254b5930.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <div id="head-bar">
+    <span>My Logo</span>
+    <div class="menu-icon">
+      <div></div>
+      <div></div>
+      <div></div>
+    </div>
+  </div>
+
+  <main>
+    <header>
+      <h1>
+        I&rsquo;m Mariane<br>
+        Glad to see you!
+      </h1>
+      <p>
+        I&rsquo;m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don&rsquo;t hesitate to contact me.
+      </p>
+      <span class="emphasize">LET&rsquo;S CONNECT</span>
+      <ul class="social-links">
+        <li class="twitter">
+          <a href="#"><i class="fa-brands fa-twitter"></i></a>
+        </li>
+        <li class="linkedin">
+          <a href="#"><i class="fa-brands fa-linkedin-in"></i></a>
+        </li>
+        <li class="medium">
+          <a href="#"><i class="fa-brands fa-medium"></i></a>
+        </li>
+        <li class="github">
+          <a href="#"><i class="fa-brands fa-github"></i></a>
+        </li>
+        <li class="angellist">
+          <a href="#"><i class="fa-brands fa-angellist"></i></a>
+        </li>
+      </ul>
+    </header>
```

Comment:
- [ ] The whole headline is a `<section>`. Kindly change that tag, the `<header>` is the toolbar as mentioned in the Project Requirements.